examples/esp32spi/minimqtt_simpletest_esp32spi.py

```diff
@@ -91,10 +91,9 @@ def message(client, topic, message):
 
 # Set up a MiniMQTT Client
 mqtt_client = MQTT.MQTT(
-    broker=os.getenv("broker"),
-    port=os.getenv("port"),
-    username=os.getenv("username"),
-    password=os.getenv("password"),
+    broker="io.adafruit.com",
```

Comment:
This example is actually meant to be optionally another server and port (just as it's simple test we might as well show the optional arg, you can use 1883 for insecure or 8883 for secure), as all it does it test callbacks and no IO specific behaviour.